bakerydemo/base/models.py

```diff
@@ -359,10 +328,10 @@ class FormField(AbstractFormField):
     """
     Wagtailforms is a module to introduce simple forms on a Wagtail site. It
     isn't intended as a replacement to Django's form support but as a quick way
-    to generate general purpose data-collection form without having to write
+    to generate a general purpose data-collection form without having to write
```

Comment:
"...to generate a general purpose data-collection form..." is a direct quote from the docs. Equally, unless I've misunderstood the functionality, it'll only send an email that the editor/ admin has explicitly set? e.g. it can't be used as a form to send emails to a list of arbitrary emails, which I think "email-sending form" may imply to a user.